deepchem/models/keras_model.py

```diff
@@ -302,16 +303,20 @@ def fit(self,
     callbacks: function or list of functions
       one or more functions of the form f(model, step) that will be invoked after
       every step.  This can be used to perform validation, logging, etc.
+    all_losses: list, optional (default False)
```

Comment:
Should be "default None".  Also we can give a more specific type: `Optional[List[float]]`.

Reply:
Good suggestion, will fix!